examples/05_graphical_pipelines.ipynb

```diff
@@ -0,0 +1,5146 @@
+    "## How to build a Graphical Pipeline\n",
+    "Two ways to build a graphical pipeline:\n",
+    "\n",
+    "1. Pass all steps to the pipeline during initialisation as for the sequential pipelines.\n",
+    "\n",
+    "```python\n",
+    "pipeline = Pipeline([\n",
+    "    {\"skobject\": skobject1, \"name\": \"bar\", \"edges\": {\"X\": \"y\"}},\n",
+    "    {\n",
+    "        \"skobject\": skobject2,\n",
+    "        \"name\": \"foo\",\n",
+    "        \"edges\": {\"X\": \"X\", \"y\": \"bar\"},\n",
+    "    },\n",
+    "    {\n",
+    "        \"skobject\": skobject1,\n",
+    "        \"name\": \"bar_inverse\",\n",
+    "        \"edges\": {\"X\": \"foo\"},\n",
+    "        \"method\": \"inverse_transform\",\n",
+    "    },\n",
+    "])\n",
+    "```\n",
+    "\n",
+    "2. Create a pipeline object and add the steps one by one.\n",
+    "\n",
+    "```python\n",
+    "\n",
+    "pipeline = Pipeline()\n",
+    "pipeline = pipeline.add_step(skobject1, \"bar\", edges={\"X\": \"y\"})\n",
+    "pipeline = pipeline.add_step(skobject2, \"foo\", edges={\"X\": \"X\", \"y\": \"bar\"})\n",
+    "pipeline = pipeline.add_step(\n",
+    "    skobject1, \"bar_inverse\", edges={\"X\": \"foo\"}, method=\"inverse_transform\"\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "The `add_step`'s parameter or key of the dicts in the step list during initialisation are:\n",
+    "\n",
+    "* skobject: The sktime object added to the pipeline\n",
+    "* name: The name of the step\n",
+    "* edges: The keys of the dictionary indicate the input of the skobject (X or y), and the values are the names of the steps that should be connected to the input argument. Note subsetting using `__` and feature union via lists are supported.\n",
+    "* method: The skobject's method that should be called. If not provided, the default method would be inferred based on the added skobject. This parameter is used for the inverse_transform method. Optional.\n",
+    "* kwargs: Additional keyword arguments passed to the sktime object. Optional.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "## Examples\n",
+    "### Forecasting Pipeline\n",
+    "A simple forecasting pipeline implemented using the graphical pipeline.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/benediktheidrich/code/sktime/sktime/pipeline/pipeline.py:160: UserWarning: This generalised graphical pipeline is experimental, with all the usual risks of edge features. For mature alternatives, use single-purpose pipelines and compositors, such as TransformedTargetForecaster, ForecastingPipeline, ClassificationPipeline, etc., see for instance notebooks 01_forecasting.ipynb and 02_classification.ipynb athttps://github.com/sktime/sktime/blob/main/examples/.\n",
```

Comment:
this causes a lot of warnings. To you want to deactivate them at the start of the nb? you can also do so via `set_config(warnings="off")`

Reply:
Done